### ISSUE TITLE
Custom admin page accessibility scans

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9462]
# Motivation

We need to be compliant with the DOJ accessibility requirement by April 24, 2026. This includes administrative pages and pages behind authentication. We have identified and created a list of all admin routes as a first step in https://github.com/uiowa/uiowa/issues/9437.

We should scan these pages to identify any compliance issues so we can address them in order to achieve compliance.

# Additional Info

Spreadsheet: https://iowa.sharepoint.com/:x:/r/sites/CSI-Drupal2/Shared%20Documents/a11y/routes-custom.xlsx?d=wdc668ce4b78245f48757a96ea585d91a&csf=1&web=1&e=q1dAYA

Previous issue: https://github.com/uiowa/uiowa/issues/9437

**Comment by briand44** [comment]
## Grooming
 
**Quadrant:** 

Clear

 **Discussion:**
 - scanning all or most of custom routes
 - may be some noise in there - not compliance related
 - document issues identified
 - deliverable - issues created for compliance items we need to address
 - siteimprove links to wcag 2.2 but still relevant in 2.1

**Proposed solution or definition of done:** 
- Scan custom routes identified in https://github.com/uiowa/uiowa/issues/9437 with Siteimprove
- document findings from scans
- audit findings and create issues for compliance related issues (A&AA)

**Comment by pyrello** [comment]
A few things:
- I think this is unblocked. I'm still doing a bit of tinkering on the spreadsheet, but I think someone could start working on this issue now.
- I haven't had a chance to test the code snippet above.

**Comment by joewhitsitt** [comment]
Thanks @pyrello for the quick start snippet here.

**Comment by joewhitsitt** [comment]
Here are a few "global" backend items that didn't seem to fit in the route specific spreadsheet:

# Container element is empty

- block-claro-primary-local-tasks
- ck-body-wrapper

# Color contrast does not meet enhanced requirement

- buttons/links within toolbar

# Interactive element does not meet minimum size nor spacing

- CKEditor buttons, dropbuttons and the "About text formats" link.

# Page does not start with a level 1 heading

- It does, but it doesn't like the placement I guess?

# Text not included in an ARIA landmark

- drupal-live-announce div

# Field input error is not announced in full

- CKEditor live announce
